source/extensions/filters/http/compressor/compressor_filter.cc

```diff
@@ -0,0 +1,26 @@
+#include "extensions/filters/http/compressor/compressor_filter.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace Compressor {
+
+CompressorFilterConfig::CompressorFilterConfig(
+    const envoy::extensions::filters::http::compressor::v3::Compressor& generic_compressor,
+    const std::string& stats_prefix, Stats::Scope& scope, Runtime::Loader& runtime,
+    Compression::Compressor::CompressorFactoryPtr compressor_factory)
+    : Common::Compressors::CompressorFilterConfig(
+          generic_compressor,
+          stats_prefix + "compressor." + generic_compressor.stats_prefix() +
```

Comment:
now that #11105 merged, we can delete the `stats_prefix` field, and use generic_compressor.compressor_library().typed_config().name().